spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsTransmitter.java

```diff
@@ -108,6 +108,10 @@ private void transmitToYggAdmin(eu.coatrack.api.Metric metricToTransmit) {
             Object idOfTransmittedMetric = restTemplate.postForObject(uriToTransmitMetric, metricToTransmit, Long.class);
             log.info("transmitted Metrics to admin: {} - response was metric ID {}", metricToTransmit.toString(), idOfTransmittedMetric);
 
+            if(metricToTransmit.getHttpResponseCode() == 503){
+                log.error("An unexpected error occurred when your CoatRack gateway contacted the central CoatRack portal in order to verify an API key that was sent by a client: Service is not unavailable/accessible, please check if the service is accessible and gateway config in https://coatrack.eu/admin/proxies");
```

Comment:
Please move this message to the method that calls CoatRack Admin to check the API key.

Reply:
Ok, I found it, and as you said there was already a log.error and exception on the code, I just changed the message.
I already committed and pushed the code